code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -365,10 +369,12 @@
 	var/mob/living/carbon/xenomorph/xeno_ruler = owner
 	if(!(selected_xeno.xeno_caste.caste_flags & CASTE_CAN_BE_LEADER))
 		if(feedback)
-			to_chat(xeno_ruler, span_xenowarning("This caste is unfit to lead."))
+			xeno_ruler.balloon_alert(xeno_ruler, "Xeno cannot lead")
 		return
 	if(feedback)
+		xeno_ruler.balloon_alert(xeno_ruler, "Xeno promoted")
 		to_chat(xeno_ruler, span_xenonotice("We've selected [selected_xeno] as a Hive Leader."))
+		selected_xeno.balloon_alert(selected_xeno, "Promoted to leader")
 		to_chat(selected_xeno, span_xenoannounce("[xeno_ruler] has selected us as a Hive Leader. The other Xenomorphs must listen to us. We will also act as a beacon for the Queen's pheromones."))
```

Comment:
remove at minimum the first to chat, you could consider the second one as important to explain what it does to new players